requirements/prod.in

```diff
@@ -7,6 +7,6 @@ PyYAML
 click
 python-dateutil
 gvgen
-pydantic
+pydantic<2.0.0
```

Comment:
What errors/issues were you hitting with v2?

Reply:
Right. Pydantic v2 [includes a v1](https://docs.pydantic.dev/2.2/migration/#continue-using-pydantic-v1-features) API, so it may be possible to resolve these by importing from `pydantic.v1`. There's also a migration tool that can help.

That said, [we have a year](https://docs.pydantic.dev/2.0/version-compatibility/#:~:text=Active%20development%20of%20V1%20has%20already%20stopped%2C%20however%20critical%20bug%20fixes%20and%20security%20vulnerabilities%20will%20be%20fixed%20in%20V1%20for%20one%20year%20after%20the%20release%20of%20V2.), so this is fine for now.